verisignmdns/src/main/java/denominator/verisignmdns/VerisignMDNSProvider.java

```diff
@@ -0,0 +1,132 @@
+package denominator.verisignmdns;
+
+import denominator.BasicProvider;
+
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+import javax.inject.Singleton;
+
+import dagger.Provides;
+import denominator.AllProfileResourceRecordSetApi;
+import denominator.CheckConnection;
+import denominator.DNSApiManager;
+import denominator.ResourceRecordSetApi;
+import denominator.ZoneApi;
+import denominator.config.GeoUnsupported;
+import denominator.config.NothingToClose;
+import denominator.config.WeightedUnsupported;
+
+import denominator.verisignmdns.VrsnMdnsContentHandler.RecordListHandler;
+import denominator.verisignmdns.VrsnMdnsContentHandler.ZoneListHandler;
+import denominator.verisignmdns.VrsnMdnsErrorDecoder.VrsnMdnsError;
+import feign.Feign;
+import feign.codec.Decoder;
+import feign.codec.Encoder;
+import feign.codec.ErrorDecoder;
+import feign.sax.SAXDecoder;
+
+public class VerisignMDNSProvider extends BasicProvider {
+    private final String url;
+
+    public VerisignMDNSProvider() {
+        this(null);
+    }
+
+    /**
+     * @param url
+     *            if empty or null use default
+     */
+    public VerisignMDNSProvider(String url) {
+        this.url = url == null || url.isEmpty() ? "https://api.dns-tool.com/dnsa-ws/V2.0/dnsaapi" : url;
+    }
+
+    @Override
+    public String url() {
+        return url;
+    }
+
+    @Override
+    public Map<String, Collection<String>> credentialTypeToParameterNames() {
+        Map<String, Collection<String>> options = new LinkedHashMap<String, Collection<String>>();
+        options.put("password", Arrays.asList("username", "password"));
+        return options;
+    }
+
```

Comment:
unless all basic record types are in fact supported, you may want to override similar to below, noting which documentation on Verisign's website can be referred to for updates.

https://github.com/Netflix/denominator/blob/master/ultradns/src/main/java/denominator/ultradns/UltraDNSProvider.java#L70